pkg/tempopb/trace_utils.go

```diff
@@ -0,0 +1,28 @@
+package tempopb
+
+import (
+	"strings"
+
+	"github.com/gogo/protobuf/jsonpb"
+)
+
+// Marshal a Trace to an OTEL compatible JSON replacing resourceSpans by batches
+func MarshalToJSONV1(t *Trace) ([]byte, error) {
```

Comment:
it would be nice to get a short comment about why this method exists. how we renamed the batches field to resourceSpans and these utility methods are for (un)marshaling json from the v1 traces endpoint to the current proto object

Reply:
Sure, I can add more info